src/WorkspacesManagementService.php

```diff
@@ -69,14 +70,19 @@ public function deleteWorkspacesAndConfigurations(string $configurationId): void
         $workspaces = new Workspaces($this->sapiClient);
         $configurationWorkspaces = $this->getConfigurationWorkspaces($configurationId);
         if (!empty($configurationWorkspaces)) {
-            $workspaces->deleteWorkspace($configurationWorkspaces[0]['id']);
+            foreach ($configurationWorkspaces as $configurationWorkspace) {
+                $workspaces->deleteWorkspace($configurationWorkspace['id']);
+            }
         }
         $configuration = $components->getConfiguration(
             self::SANDBOXES_COMPONENT_ID,
             $configurationId
         );
         if (!empty($configuration['configuration']['parameters']['id'])) {
-            $this->sandboxesClient->delete((string) $configuration['configuration']['parameters']['id']);
+            try {
+                $this->sandboxesClient->delete((string) $configuration['configuration']['parameters']['id']);
+            } catch (Throwable $e) {
+            }
```

Comment:
Tady jsem upravil mazani, aby to smazalo vsechno ;) Protoze se obcas stane, ze ti testy spadnou a zustanou tam viset workspacei nebo sandboxy, vic nez jeden per konfiguraci

Reply:
Teardown by se měl pustit i když test spadne, mně to fungovalo vždycky dobře, pokud si nepustil najednou CI a lokálně testy, nebo ti neběžely dvě pipelines paralelně. Ale jestli to takhle bude spolehlivější tak super.